src/main/groovy/com/avioconsulting/mule/linter/model/Application.groovy

```diff
@@ -6,25 +6,33 @@ class Application {
     static final String POM_FILE = 'pom.xml'
 
     File applicationPath
-    List<ProjectFile> files = []
+    List<PropertyFile> propertyFiles = []
     PomFile pomFile
+    String name
 
     Application(File applicationPath) {
         this.applicationPath = applicationPath
         if (!this.applicationPath.exists()) {
             throw new FileNotFoundException( APPLICATION_DOES_NOT_EXIST + applicationPath.absolutePath)
         }
         pomFile = new PomFile(applicationPath, POM_FILE)
+        this.name = pomFile.getArtifactId()
+
+        File resourcePath = new File(applicationPath, 'src/main/resources')
+        if (!resourcePath.exists()) {
+            throw new FileNotFoundException( APPLICATION_DOES_NOT_EXIST + resourcePath.absolutePath)
+        }
+        resourcePath.eachDirRecurse { dir ->
+            dir.eachFileMatch(~/.*.properties/) { file ->
+                propertyFiles.add(new PropertyFile(file))
+            }
+        }
```

Comment:
Refactor to a method for loading properties, keep constructor a simple set of method calls.  Easier to unit test that portion as a method then